trinity/config.py

```diff
@@ -176,7 +177,9 @@ def database_dir(self) -> Path:
 
         This is resolved relative to the ``data_dir``
         """
-        if self.sync_mode == SYNC_FULL:
+        if self.sync_mode == SYNC_FAST:
+            return self.data_dir / DATABASE_DIR_NAME / "fast"
```

Comment:
Hm, this is unfortunate. It will cause everyone not on a light client to re-sync from scratch :-/

I'm not sure that we even need a separate DB dir for fast vs full, but maybe that's just me not wanting to trigger a resync for everyone (without manual intervention of moving their data directory).

Counter-argument: it's alpha software, suck it up testers :P

I guess I land on: we keep this change, but add a big warning to users in the next release notes saying that they should move their data dir if they don't want to resync.